<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>donationswindow</class>
 <widget class="QMainWindow" name="donationswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(96, 195, 244);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tab_add">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1111</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tab">
     <attribute name="title">
      <string>Add/Manage donations</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(133, 214, 255);</string>
      </property>
      <property name="title">
       <string>Add donation</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Category name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Amount of money</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantity</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Phone number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_name">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>190</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_quantity">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_description">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>300</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_amount">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>350</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>480</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_phonenumber">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>410</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>510</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QComboBox" name="le_categoryname">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>clothes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blankets</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>food</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>money</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton5">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>420</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>verify</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>671</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(133, 214, 255);</string>
      </property>
      <property name="title">
       <string>List of donations</string>
      </property>
      <widget class="QTableView" name="tab_donation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>651</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pdf">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>370</y>
         <width>111</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate PDF</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sortbox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>410</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Sort by ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sort by category name</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>410</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_sort">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>410</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_catalogue">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>410</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>catalog</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>560</y>
        <width>421</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(133, 214, 255);</string>
      </property>
      <property name="title">
       <string>Delete</string>
      </property>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_idsupp">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>440</y>
        <width>671</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(133, 214, 255);</string>
      </property>
      <property name="title">
       <string>Search and sort</string>
      </property>
      <widget class="QLineEdit" name="le_search">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>300</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>300</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QComboBox" name="searchbox">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>300</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Search by category name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Search by ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Search by name</string>
        </property>
       </item>
      </widget>
      <widget class="QTableView" name="tab_donation2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>661</width>
         <height>271</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_search">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>300</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>670</y>
        <width>421</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(133, 214, 255);</string>
      </property>
      <property name="title">
       <string>generate pdf</string>
      </property>
      <widget class="QPushButton" name="stats">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>stats</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Generate invoice</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>471</width>
        <height>611</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  white;
border-radius:35px;

</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_77">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color:#341763;
color:#4a5bcf;
font-size:20px;</string>
       </property>
       <property name="text">
        <string>Invoice</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_81">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Total collects  money</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_96">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Total collects budget</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_102">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>250</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Total services budget</string>
       </property>
      </widget>
      <widget class="Line" name="line_6">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>50</y>
         <width>451</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(222, 226, 230, 0.8);
border-width: 1px;
border: 2px solid rgba(222, 226, 230, 0.8);
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_104">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>430</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Donation money NET </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_79">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>330</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Total donations money</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_total_service_budgets">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>290</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>***************************</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_total_collections">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>***************************</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_total_donations">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>380</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>***************************</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_total_collect_budgets">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>***************************</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_net">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>470</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>*********</string>
       </property>
      </widget>
      <widget class="QPushButton" name="le_invoice">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>530</y>
         <width>211</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
border-color: rgb(85, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Generate invoice</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_print">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>570</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);
border-color: rgb(85, 0, 0);
</string>
       </property>
       <property name="text">
        <string>PRINT</string>
       </property>
      </widget>
      <widget class="Line" name="line_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>500</y>
         <width>451</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(222, 226, 230, 0.8);
border-width: 1px;
border: 2px solid rgba(222, 226, 230, 0.8);
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Delete donation</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuapp">
    <property name="title">
     <string>general</string>
    </property>
    <addaction name="actionabout_this_app"/>
   </widget>
   <addaction name="menuapp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionemployee"/>
   <addaction name="actiondestitutes"/>
   <addaction name="actiondonations"/>
   <addaction name="actionservices"/>
   <addaction name="actionevents"/>
  </widget>
  <action name="actionemployee">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/xxxxxxxxx-01.png</normaloff>:/images/xxxxxxxxx-01.png</iconset>
   </property>
   <property name="text">
    <string>employee</string>
   </property>
   <property name="toolTip">
    <string>employee</string>
   </property>
  </action>
  <action name="actiondestitutes">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Untitled-1-01.png</normaloff>:/images/Untitled-1-01.png</iconset>
   </property>
   <property name="text">
    <string>destitutes</string>
   </property>
   <property name="toolTip">
    <string>destitutes</string>
   </property>
  </action>
  <action name="actiondonations">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/sdf1-01.png</normaloff>:/images/sdf1-01.png</iconset>
   </property>
   <property name="text">
    <string>donations</string>
   </property>
   <property name="toolTip">
    <string>donations</string>
   </property>
  </action>
  <action name="actionservices">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/xxxxxxccccccccccccc-01.png</normaloff>:/images/xxxxxxccccccccccccc-01.png</iconset>
   </property>
   <property name="text">
    <string>services</string>
   </property>
   <property name="toolTip">
    <string>services</string>
   </property>
  </action>
  <action name="actionevents">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/qsdfg-01.png</normaloff>:/images/qsdfg-01.png</iconset>
   </property>
   <property name="text">
    <string>events</string>
   </property>
   <property name="toolTip">
    <string>events</string>
   </property>
  </action>
  <action name="actionabout_this_app">
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
